tests/layouts/CoreLayout.spec.js

```diff
@@ -34,11 +34,4 @@ describe('(Layout) Core', function () {
     expect(_component.type).to.equal('div')
   })
 
-  it('Should render a child component.', function () {
```

Comment:
@davezuko yes all transforms are working, see below as usual. i also tested hmr with multiple nested `AboutView`s to be sure react-transform is walking the full component tree :+1: 

![hmr-stateless](https://cloud.githubusercontent.com/assets/1539088/11770941/d5c43c8c-a1d7-11e5-97ea-14e6ecc5de60.gif)